### DISCUSSION TITLE
What size should my corpus be?

### DISCUSSION THREAD

**Discussion by moonman239** [explosion/spaCy#10974] [Help: Coding & Implementations] [answered]
I'm thinking of building a corpus based on social media posts that are written in English. How large should the corpus be so that I can get a good resulting spaCy model?

**Comment by polm** [comment, marked as answer]
It depends, but bigger is (almost always) better. There is no magic number or way to answer this perfectly in advance, but at a minimum you would need a few hundred examples, and if possible it's better to get thousands. [This flowchart](https://support.prodi.gy/t/annotation-flowchart-named-entity-recognition/1556) may be helpful.

For more perspective, what kind of model are you trying to train, and what are you trying to label with it specifically?
